taipy/core/config/job_config.py

```diff
@@ -33,8 +33,13 @@ class JobConfig:
     _DEFAULT_MODE = "standalone"
 
     _MODE_TO_MODULE: Dict[str, str] = {
-        "airflow": "taipy.airflow.scheduler",
-        "enterprise": "taipy.enterprise.core.scheduler.scheduler",
+        "airflow": "taipy.airflow",
+        "enterprise": "taipy.enterprise",
+    }
+
+    _MODE_TO_SCHEDULER_PATH: Dict[str, str] = {
+        "airflow": ".scheduler",
+        "enterprise": ".core.scheduler.scheduler",
     }
```

Comment:
We can add _MODE_TO_CONFIG_PATH mapping and refactor the logic in _get_config_cls too.

Reply:
imo, the current version of _get_config_cls is good already, the reason the need for mode to scheduler is because enterprise and airflow version has different base path for config. What do you think? :D

Reply:
It is good already but in the future there may be cases where the Config class is not in the root of the package so it is better to have explicit path to the Config class. What do you think?